integration/image/list_test.go

```diff
@@ -56,6 +56,53 @@ func TestImagesFilterMultiReference(t *testing.T) {
 	}
 }
 
+func TestImagesFilterUntil(t *testing.T) {
+	ctx := setupTest(t)
+
+	client := testEnv.APIClient()
+
+	name := strings.ToLower(t.Name())
+
+	imgs := make([]string, 5)
+	var olderUntil string
+	var laterUntil string
+	for i := 0; i < 5; i++ {
+		if i > 0 {
+			time.Sleep(time.Millisecond)
+		}
+		ctrName := fmt.Sprintf("%s%d", name, i)
+		ctr := container.Create(ctx, t, client, container.WithName(ctrName))
+		id, err := container.CommitOptions(ctx, ctr, types.ContainerCommitOptions{Reference: ctrName})
```

Comment:
Sorry about that, I completely botched my earlier suggestion. Let's try this again:
```suggestion
		id, err := client.ContainerCommit(ctx, ctr, container.CommitOptions{Reference: ctrName})
```